lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -35,13 +35,13 @@ module.exports = [
         extendedInfo: {
           value: {
             results: {
-              length: 18,
```

Comment:
i kinda like being so explicit, though ideally in failure we see exactly where the difference is.

can we avoid dropping back to `>15` ?

Reply:
:( alright